src/Cli/Microsoft.DotNet.Cli.Utils/NativeMethods.cs

```diff
@@ -63,6 +76,14 @@ internal struct JobObjectExtendedLimitInformation
 
             [DllImport("kernel32.dll", SetLastError = true)]
             internal static extern bool AssignProcessToJobObject(IntPtr hJob, IntPtr hProcess);
+
+            [DllImport("kernel32.dll", CharSet = CharSet.Auto)]
+            [DefaultDllImportSearchPaths(DllImportSearchPath.System32)]
```

Comment:
None of the existing PInvokes in this file have DefaultDllImportSearchPaths. Do you really need it? (The search path is typically set at the assembly level.)

Reply:
The parent process ID method call is used from an elevated instance, so I started adding it. I can move it to the assembly level. Does that impact anything on non-Windows for the assembly?

Reply:
Ok, it is probably safer to just add it on the few APIs that you adding, given that this is RC fix.